aio/aio-proxy/aio_proxy/parsers/code_commune.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional
+
+from aio_proxy.labels.helpers import codes_communes
+
+
+def validate_code_commune(code_commune_clean: str) -> Optional[str]:
+    """Check the validity of code_commune.
+
+    Args:
+        code_commune_clean(str, optional): code commune extracted and cleaned.
+
+    Returns:
+        None if code_commune_clean is None.
+        code_commune_clean if valid.
+
+    Raises:
+        ValueError: if code_commune_clean not valid.
+    """
+    if code_commune_clean is None:
+        return None
+    if len(code_commune_clean) != 5:
+        raise ValueError("Code commune doit contenir 5 caractères !")
+    if code_commune_clean not in codes_communes:
```

Comment:
Not sure what is the most efficient here.
I believe a hash would ve better than searching in an array

Reply:
[a Simple regex is doing the work](https://github.com/etalab/schema-irve/blob/a74411123830283ae4ee4d4002455b54196280bb/schema.json#L205)